docs/content/latest/reference/drivers/ysql-client-drivers.md

```diff
@@ -177,10 +177,27 @@ For a tutorial on building a sample Python application that uses `psycopg2`, see
 
 #### Install the psycopg2 binary
 
-To install the `psycopg2` binary package, run the following pip install` command:
+To install the `psycopg2` package, run the following `pip install` command:
 
 ```sh
-$ pip3 install psycopg2
+$ pip install psycopg2
+```
+
+### aiopg
+
+[aiopg](https://aiopg.readthedocs.io/en/stable/) is a library for accessing a PostgreSQL database from the asyncio (PEP-3156/tulip) framework. It wraps
```

Comment:
from the asyncio framework
or
using the asyncio framework

Reply:
changed to "using"